bounswe/bounswe2017group5
# Design a new logo and add it to our pages

## About

* **Related Issues:** https://github.com/bounswe/bounswe2017group5/issues/89

As discussed in the issue above, I designed a logo and put it on all our pages - including a big one on the homepage.

### Changes

- Design a new logo and add 2 versions of it to our assets: orange (`logo.png`) and white (`logo-white.png`). They both have transparent backgrounds.
- Add our logo to the navbar and homepage.

## Visuals
**Note:** There's been a minor change on these visuals: Our logo color is now `#FF9500`, which is our primary color. It is same as what you see on navbar and button backgrounds.

### Desktop
![screencapture-file-users-smddzcy-bounswe2017group5-fe-index-html-1508965743144](https://user-images.githubusercontent.com/13895224/32023641-8b2ab12c-b9e2-11e7-94ee-1b575aa2b6a7.png)

### Mobile (iPhone 6 screen size)
![screencapture-file-users-smddzcy-bounswe2017group5-fe-index-html-1508966104242](https://user-images.githubusercontent.com/13895224/32023694-bb34aa12-b9e2-11e7-831e-5475ec97c0a9.png)


## Testing

- Open homepage in different browsers that we support (last 2 versions of Chrome and Firefox, IE 11, Safari etc.)
- Check whether logos look properly in different screen sizes on all those browsers.
